src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $week-day, $index in $start-day {
+    &--start-day-#{$week-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
The expression '$cellSize * $index + $index' may not work as expected due to operator precedence. If you want to add $index after multiplying $cellSize by $index, you should use parentheses: '($cellSize * $index) + $index'.